tests/nightly/gpu/test_bb2.py

```diff
@@ -88,8 +88,9 @@ def test_retrieval_none(self):
         _test_bb2_rag(KnowledgeAccessMethod.NONE, n_docs=1)
 
 
-@testing_utils.skipUnlessGPU
-@unittest.skipIf(LOCAL, "Skipping Test because its slow and mem intensive")
+# @testing_utils.skipUnlessGPU
```

Comment:
you may have left these comments in by mistake

Reply:
I'm just testing to right now to see if skipping these tests fixes the cache issue. If it does, then yea will remove this comment, but if it doesn't, then I'll revert these changes related to skipping tests

Reply:
(And address the cache issue in a separate pr)